api/models/node.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 type NodeStatus struct {
-	Primary bool `json:"primary" form:"primary" binding:"Required"`
+	Primary string `json:"mode" form:"mode" binding:"Required"`
```

Comment:
@replay doesn't this change the http parameter from primary to mode? why this change?